content/blog/2021-07-03-darktable-3.6/index.md

```diff
@@ -131,22 +131,22 @@ The graph represents color "volume" by rendering the more frequently used colors
 
-This tool, originally used in video editors, is useful for color-matching different images in a series by showing what the dominant colors are. This means that users only have to ensure the consistency of dominant colors between frames. It can also be used to analyse, teach and further understand the effects of module settings on image colors.
+This tool, originally used in video editors, is useful for color-matching different images in a series by showing what the dominant colors are. This means that users only have to ensure the consistency of dominant colors between frames. It can also be used to analyze, teach and further understand the effects of module settings on image colors.
 
 Some things that the vectorscope cannot do:
 
 - The hue ring is not a gamut check, as a color can be within the hue ring, yet out of gamut due to its darkness/lightness.
 - When adjusting an image based upon a color checker, faster and more accurate results will come from using the new "calibrate with a color checker" option in *color calibration* ([see above](#Color-calibrationnbspadd-color-checker-support)).
-- The vectorscope does not have a "skin tone line." Skin tone varies in chromaticity, hence any such line would inevitably represent a presupposition. Furthermore (see next point) this scope does not provide specific guidance to correct for specific image content.
+- The vectorscope does not have a "skin tone line." Skin tone varies in chromaticity, hence any such line would inevitably represent a presupposition. Furthermore (see next point), this scope does not provide specific guidance to correct for specific image content.
 - The vectorscope represents a colorimetric encoding of an image, which inevitably diverges from a viewer's perception of the image.
 
 ## Darkroom UI Amendments
 
 The header line of processing modules has been amended to include a mask indicator, for those modules where a drawn or parametric mask has been used. You can click on this icon to show or hide the mask.
 
-A number of processing modules include drawn elements (e.g. the charts in _filmic rgb_ or the spline curves in the _contrast equalizer_). These elements default to a 16:9 aspect ratio but can be resized by hovering your mouse over them and scrolling with the mouse's scroll wheel while holding Ctrl.
+A number of processing modules include drawn elements (e.g., the charts in _filmic rgb_ or the spline curves in the _contrast equalizer_). These elements default to a 16:9 aspect ratio, but can be resized by hovering your mouse over them and scrolling with the mouse's scroll wheel while holding Ctrl.
```

Comment:
US style guides supposedly require the comma; British do not. https://jakubmarian.com/comma-after-i-e-and-e-g/

Reply:
ew no - write as "for example," if you're gonna do that I think